statler_api/views.py

```diff
@@ -13,20 +15,19 @@ def healthCheck(request):
     return HttpResponse("The API is alive.")
 
 
-def getPlayList(request):
+def getPlayList(request, play_list_id):
     """called when a GET request is sent to /api/play-list/
     returns json of the list of plays
     see https://docs.djangoproject.com/en/1.8/topics/serialization/"""
-    #TODO: get an actual PlayList object instead of all plays
 
-    aPlayDictList = [vars(Play(x)) for x in PlayDAO.objects.all()]
+    aPlayEntryList = PlayList(get_object_or_404(PlayListDAO, url_title=play_list_id)).entries
 
-    # Remove reviews from each play dict, because we aren't going
-    # that deep in the object graph
-    for aPlayDict in aPlayDictList:
-        del aPlayDict["reviews"]
-        
-    return HttpResponse(json.dumps(aPlayDictList))
+    aPlayEntryList = [vars(x) for x in aPlayEntryList]
+    for aPlayEntry in aPlayEntryList:
+        aPlayEntry["play"] = vars(aPlayEntry["play"])
+        del aPlayEntry["play"]["reviews"]
```

Comment:
@B3nD3lan3y, this is a fixed play list endpoint. The serialization is messy and will be moved into the classes by you. But the pattern of using vars() recursively might actually be useful